lamb/src/main/java/com/scalar/kelpie/lamb/ContractConfigManager.java

```diff
@@ -0,0 +1,55 @@
+package com.scalar.kelpie.lamb;
+
+import com.scalar.kelpie.config.Config;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+public class ContractConfigManager {
+  private static final String TEST_CONFIG_TABLE = "benchmark_config";
+  private static final String CONTRACT_CONFIG = "contract_config";
+  private static final String VARIABLE_CONFIG = "variable_config";
+  private static final String CLASS_FILE = "class_file";
+  private static final String PROPERTIES = "properties";
+  private static final String ARGUMENTS = "arguments";
+
+  private final JsonObject contractConfig;
+  private final JsonObject variableConfig;
+
+  public ContractConfigManager(Config config) throws IOException, FileNotFoundException {
```

Comment:
FileNotFoundException is not needed because there is a more general exception IOException in the throws list already.